src/pages/install/index.md

```diff
@@ -4,25 +4,46 @@ import TabItem from '@theme/TabItem';
 
 # Installing RGBDS
 
-Most people will want to use a release (the latest the bestest!), but if you are willing to help us test new features, you may consider [using a development version](/install/master).
+Most people will want to use a stable release (the more recent the better). 
 
-The two main options to install a release of RGBDS are to download pre-built executables, or to [build from source](/install/source).
-Below is a list of <abbr title="Operating System">OS</abbr>es for which we know pre-built executables are available.
+Below is a list of Operative Systems for which pre-built executables are available. If none of these options fits your needs, you can [build from source](/install/source).
 
```

Comment:
```suggestion
Below is a list of OSes for which pre-built executables are available.
If none of these options fits your needs, you can [build from source](./source.md).
```
We can use `<abbr title="Operating Systems">` to explain "OSes" instead, if desired.
Additionally, linking to the Markdown file makes it checked by Docusaurus, which improves reliability.

Reply:
Agree!